lib/rules/custom-property-empty-line-before/index.js

```diff
@@ -24,7 +23,8 @@ const messages = ruleMessages(ruleName, {
 	rejected: 'Unexpected empty line before custom property',
 });
 
-function rule(expectation, options, context) {
+/** @type {import('stylelint').StylelintRule} */
+const rule = (expectation, options, context) => {
```

Comment:
In the previous refactoring pull requests, you've made the parameters consistently be `primary` and `secondaryOptions`. Shall we make the change here too?

Reply:
Oh, I missed them. Thank you! 0002fa05